### ISSUE TITLE
ORPHA:327   (Disorder)   Congenital factor VII deficiency

### ISSUE THREAD

**Issue by Marlies99** [orphanet-rare-diseases-issues/RD-CODE#8]
Hi 
Similar question here: Can the orphacode **ORPHA:327   (Disorder)   Congenital factor VII deficiency**  only be set for homozygous or compound heterozygous mutation in the gene encoding coagulation factor-7 ? 

I have a patient who has a heterozygous factor VII deficiency with 36% residual activity. How to code? Would it be correct to code it with ORPHA:327?

The linked OMIM Number is 227500:
TEXT
A number sign (#) is used with this entry because factor VII deficiency is caused by homozygous or compound heterozygous mutation in the gene encoding coagulation factor-7 (F7; 613878) on chromosome 13q34.

Thanks for your feedback.

Best regards
Marlies

**Comment by houda-ali** [comment]
Dear Marlies,
This is a good question because the text indeed seems to include both symptomatic and asymptomatic/heterozygous patients in the scope of the ORPHAcode, but we need to clarify whether this representation really makes sense, since a Disorder is supposed to be defined as an entity with homogeneous clinical presentation and management, which I'm not sure is the case here (symptomatic cases probably need specific management and follow-up, and maybe even specific statistical analyses within information systems/registries). Our team will review this entity and we will come back to you as soon as we have validated a decision.
Thank you,
Houda

**Comment by caterina-lucano-orphanet** [comment]
Dear Marlies,
Thank you for your patience. We have finished reviewing this entity and we have decided to add an autosomal dominant mode of inheritance to this disease next to the autosomal recessive (it will be visible online in the next couple of days).
This is because the disease should be delineated clinically on the residual activity of the factor and on the severity of the phenotype, and not on the genetic factor.

According to PMID:30510462 the International Society on Thrombosis and Haemostasis has reclassified FVII deficiency as follows: severe, FVII <10%, risk of spontaneous major bleeding; moderate, FVII 10%-20%, risk of mild spontaneous or triggered bleeding; mild, FVII 20%-50%, mostly asymptomatic disease.

However, to complicate the issue, in some cases of congenital FVII deficiency, there is no direct correlation between the factor plasma levels and the severity of the disease symptoms. In fact, some people with very low levels of FVII may demonstrate mild symptoms. In general, the most severe cases are either homozygous or compound heterozygous with FVII: C levels less than 2.0% of normal, but occasionally, heterozygous carriers display hemorrhagic symptoms that can be severe in rare cases (reference PMID:30797223).

Therefore, we concluded that ORPHA:327 Congenital factor VII deficiency can also be used to code for heterozygous patients, provided that they are symptomatic and that their level of FVII is underneath the cutoff for the normal range (<50%).

Hope this helps solve your issue,
Caterina
